python/lsst/analysis/drp/plotUtils.py

```diff
@@ -81,6 +84,40 @@ def generateSummaryStats(cat, colName, skymap, plotInfo):
     return patchInfoDict
 
 
+def generateSummaryStatsVisit(cat, colName, visitSummaryTable, plotInfo):
+    """Generate a summary statistic in each patch
```

Comment:
Do you actually mean patch here, or do you mean visit or detector?

Reply:
It works on either patches or detectors, I have clarified this.